requirements-dev.txt

```diff
@@ -1,11 +1,11 @@
 -r requirements.txt
-fake-factory==0.5.0
-ipdb==0.9.3
-nsot==1.1.1
-py==1.4.26
-pytest==2.7.0
-pytest-django==2.9.1
-pytest-pythonpath==0.6
-Sphinx==1.3.6
-sphinx-autobuild==0.6.0
-sphinx-rtd-theme==0.1.9
+fake-factory~=0.5.0
+ipdb~=0.9.3
+nsot>=1.1.2,~=1.1.0
```

Comment:
I feel like we should keep this version explicit since we're tying features in the pynsot CLI to explicit NSoT releases.